hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/StorageContainerManager.java

```diff
@@ -713,9 +704,12 @@ public static boolean scmInit(OzoneConfiguration conf,
             scmStorageConfig.getLayoutVersion());
         //TODO: this check need not be done if ratis is not used.
```

Comment:
Minor: confusing, can we remove the TODO?
As we start SCM Ratis Server only when ratis is enabled, that checks makes it clear.